conf/test_netmhciipan.config

```diff
@@ -18,6 +18,6 @@ params {
     // Input data
     input = 'https://raw.githubusercontent.com/nf-core/test-datasets/epitopeprediction/testdata/sample_sheets/sample_sheet_peptides_class2.csv'
 
-    tools = 'netmhciipan'
+    tools = 'netmhciipan-3.1'
```

Comment:
How is this test running through? The given data folder path is not supported anymore: https://services.healthtech.dtu.dk/services/NetMHCIIpan-3.1/data.tar.gz